docs/en/latest/support-fips-in-apisix.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Support FIPS in APISIX
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+OpenSSL 3.0 [supports](https://www.openssl.org/blog/blog/2022/08/24/FIPS-validation-certificate-issued/)
+[FIPS](https://en.wikipedia.org/wiki/FIPS_140-2) mode. So to support
+FIPS in APISIX, we just need to compile apisix-base with OpenSSL 3.0.
+
+## Compilation
```

Comment:
This doc is named "Support FIPS in APISIX", but most of its content is only about compiling APISIX-Base.

@juzhiyuan @navendu-pottekkat 
Could you help to review this doc as the role of APISIX user? Thanks!

Reply:
The title is correct. Even if we are compiling APISIX, the end goal is to support FIPS. So it is correct. Please also add YAML front matter after the title. Refer to other documentation pages.

Reply:
> The title is correct. Even if we are compiling APISIX, the end goal is to support FIPS. So it is correct. Please also add YAML front matter after the title. Refer to other documentation pages.

Most docs only contains title. So what extra fields I need to add? Could you give a doc for reference?

Reply:
See https://github.com/apache/apisix/blob/ae400b95b91240fb867c3cedc9fa65a6b5593f36/docs/en/latest/building-apisix.md?plain=1#L4

I'm trying to add these to all pages. So if we do it from the start, it would be easier.